TheMET/ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,15 @@
+/* 
+  Localizable.strings
+  TheMET
+
+  Created by Анна Ситникова on 30/04/2023.
+  
+*/
+
+"Random Artwork Of The Day" = "Произведение дня";
```

Comment:
Using a sentence/ a word inEnglish itself as a string identifier is not a commonly used practice. The problem is that sometimes we use a same English word/phrase in different App contexts, but in other localisations each context needs its own text.
Try to use the context-dependant identifiers instead.

For example, instead of "Artwork cannot be upload Make sure you are connected to Wi-Fi or your cellural network" use "featured_artwork.loading_failed"; Instead of "Try again" use "featured_artwork.load_again_cta"; etc.

Reply:
fixed